src/slack.coffee

```diff
@@ -242,4 +245,29 @@ class SlackBot extends Adapter
     channel = @client.getChannelGroupOrDMByName envelope.room
     channel.setTopic strings.join "\n"
 
+  customMessage: (data) =>
+    channel = @client.getChannelGroupOrDMByName data.message.envelope.room
+
+    msg = []
```

Comment:
API aren't explicitly clear on the format, but everything suggest that an object is needed, so I think @evansolomon is right.